components/brave_new_tab_ui/storage.ts

```diff
@@ -10,11 +10,11 @@ import { debounce } from '../common/debounce'
 const keyName = 'new-tab-data'
 
 const defaultState: NewTab.State = {
+  textDirection: 'ltr',
```

Comment:
I'm not sure about the implications of this. can we grab this info from chromium directly?

Reply:
I left `ltr` as a default value here but we are really only getting the value when we call `window.loadTimedata.getString('textdirection')` in the NTP initializer. AFAIK this is the only method we can use to explicitly fetch the text direction in the JS - my reasoning for moving that property into redux was so we only make the call once rather than littering this across our components and probably less expensive